sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsClientSample.csproj

```diff
@@ -17,7 +17,7 @@
     <PackageReference Include="CommandLineParser" Version="2.7.82" />
```

Comment:
Feel free to just delete this sample.   The library has not been released in 3 years and no longer seems to be maintained.

Reply:
Can do, will do that one in a separate PR too